<!DOCTYPE html>
<html lang="en">
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
    <title>HOME - The DENG Lab</title>
    <meta name="description" content="home">
    <meta name="keywords" content="DENG Xin, Xin DENG, X DENG, Pseudomonas, CityU, City University of HONG KONG, BMS, Department of Biomedical Sciences, Microorganism">
    <meta name="author" content="WeitongZHANG">
    <meta http-equiv="refresh" content="30">
    <link rel = "stylesheet" type = "text/css" href = "foundation.css">
    <link rel = "stylesheet" type = "text/css" href = "style.css">
</head>
<body class="bgcolor">
<!-- HEADER AREA -->
<header class="full-width header-area">
    <div class="row header-photolocate">
        <div class="large-12 columns distr">
            <a href="https://www.cityu.edu.hk/cvmls/en/index.html/"><img class="jcc-photo" src="https://raw.githubusercontent.com/Nikaika/labweb.github.io/master/imge/logo-jcc.png" /></a>
            <a href="index.html"><img class="deng_logo" src="https://raw.githubusercontent.com/Nikaika/labweb.github.io/master/imge/logo2.png" /></a>
            <a href="https://www.cityu.edu.hk/bms/"><img class="department" src="https://raw.githubusercontent.com/Nikaika/labweb.github.io/master/imge/Department%20of%20%20Biomedical%20Sciences.png" /></a>
        </div>
    </div>

    <!-- NAVIGATION AREA -->
    <div class="full-width navigation-area navcenter">
        <div class="row navcenter">
            <div>
                <nav class="top-bar nav-background">
                    <section class="top-bar-section">
                        <ul>
                            <li><a href="index.html">HOME</a></li>
                            <li><a href="PI.html">PI</a></li>
                            <li><a href="MEMBERS.html">MEMBERS</a></li>
                            <li class="active"><a href="PUBLICATIONS.html">PUBLICATIONS</a></li>
                            <li><a href="OPPORTUNITIES.html">OPPORTUNITIES</a></li>
                            <li><a href="NEWS.html">NEWS</a></li>
                        </ul>
                    </section>
                </nav>
            </div>
        </div>
    </div>
</header>
<!-- CONTENT AREA -->
<div class="banner-wrap"></div>
<div class="full-width content-area content-background">
    <div class="row content-panel">
        <div class="large-12 columns base-font Text-Align-all">
            <span class="title-type title-font">Publications</span><span class="base-font anno">#co-first authors &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*(co)corresponding authors</span>
            <p class="light-blue text-underline google-schoar-text base-font "><strong><a class="text-color-con" href="https://scholar.google.com/citations?user=Z5SYO1IAAAAJ&hl=en" target="_blank">Google Scholar</a></strong></p>
            <h4 class="light-blue base-font marginTOP">20<span class="year-text">25</span></h4>
            <span>100. He X, Zhang Y, Xu C, Fu K, Ding Y, Zhang T, Chen T, Murero A, Wang L, Xu Y, Chen C, Yang J, Li L, Zhong C, Huang L, Deng X, Shao X, Qian G. A temperature-responsive regulator that enhances virulence in the kiwifruit canker pathogen <strong><I>Pseudomonas syringae</strong></I>pv. actinidiae. <strong><I>Comput Struct Biotechnol J</strong></I>. 2025 DOI: 10.1016/j.csbj.2025.05.017.</span>		
            <br>
            <br>
            <span>99. Xu L, Shen S, Zhang Y, Guo Z, Lu B, Huang J, Li R, Shen Y, <strong>Zhang L<sup>*</sup></strong>, <strong>Deng X<sup>*</sup></strong>. Quantitative RNA pseudouridine landscape reveals dynamic modification patterns and evolutionary conservation across bacterial species. <strong><I>eLife</strong></I>. 2025</span>		
            <br>
            <br>
           <span>98. Chen F, Sun Y, Huang J, Li T, He Z, Li J, Wang T, <strong>Hu H<sup>*</sup></strong>, <strong>Deng X<sup>*</sup></strong>. VimR is a conserved master transcription regulator controlling virulence and metabolism in <I>Pseudomonas</I>. <strong><I>Nucleic Acids Res</strong></I>. 2025</span>		
            <br>
            <br>
            <span>97. Lu B, Guo Z, Liu X, Ni Y, Xu L, Huang J, Li T, Feng T, <strong>Li R<sup>*</sup></strong>, <strong>Deng X<sup>*</sup></strong>. Comprehensive comparison of the third-generation sequencing tools for bacterial 6mA profiling. <strong><I> Nat Commun</strong></I>. 2025 DOI: 10.1038/s41467-025-59187-2</span>		
            <br>
            <br>
            <span>96. Huang J, Sun Y, Chen F, Li S, Han L, Li J, Li R, He Z, Hua C, Yao C, Li T, Lu B, Chang F,<strong>Deng X<sup>*</sup></strong>. Global transcription factors analyses reveal hierarchy and synergism of regulatory networks and master virulence regulators in <strong><I>Pseudomonas aeruginosa</strong></I>. <strong><I>eLife</strong></I>. 2025</span>		
            <br>
            <br>
            <span>95. Zhang Z, Zhao S, Zhu D, Guan M, Xiong L, He Z,Li Y, Shi Y, Xu Z, Deng X, Cui Z. Design, Synthesis, and Biological Evaluation of Asymmetrical Disulfides Based on Garlic Extract as <I>Pseudomonas aeruginosa</I> pqs Quorum Sensing Inhibitors. <strong><I> J Agri Food Chem</strong></I>. 2025 DOI: 10.1021/acs.jafc.4c12713</span>		
            <br>
            <br>
            <h4 class="light-blue base-font marginTOP">20<span class="year-text">24</span></h4>
            <span>94. Xu M, Yu S, Li P, Chen Y, Chen Y, Pan J,<strong>Deng X<sup>*</sup></strong>, <strong>Hu H<sup>*</sup></strong>. Tailored multilayer nanoparticles against resistant <I>P. aeruginosa</I> by disrupting the thickened mucus, dense biofilm and hyperinflammation. <strong><I>J Control Release</strong></I>. 2024 DOI: 10.1016/j.jconrel.2024.12.040.</span>		
            <br>
            <br>
            <span>93. Zhang Z, He Z, Shi Y, Guan M, Zhao D, Zhu D, Xiong L, Li Y, Deng X, Cui Z. Structure-Based Discovery of Symmetric Disulfides from Garlic Extract as <I>Pseudomonas aeruginosa</I> Quorum Sensing Inhibitors. <strong><I> J Agric Food Chem</strong></I>. 2024 DOI: 10.1021/acs.jafc.4c04404</span>		
            <br>
            <br>
            <span>92. Tian Y, Wang X, Huang H, Deng X, Zhang B, Meng Y, Wu L, Chen H, Zhong Y, Chen W. Genome-Wide Identification of the DnaJ Gene Family in Citrus and Functional Characterization of ClDJC24 in Response to Citrus Huanglongbing.<strong><I> Int J Mol Sci</strong></I>. 2024 DOI: 10.1016/j.cclet.2024.110044</span>		
            <br>
            <br>
            <span>91. Hua C, Huang J, Sun Y, Wang T, Li Y, Cui Z <strong>Deng X<sup>*</sup></strong>. (2024) Hfq mediates transcriptome-wide RNA structurome reprogramming under virulence-inducing conditions in a phytopathogen.<strong><I> Cell Reports</strong></I>. 2024 DOI: 10.1016/j.celrep.2024.114544</span>		
            <br>
            <br>
            <span>90. Tan L, Guo Z, Shao Y, Ye L, Wang M, <strong>Deng X</strong>, Chen S, Li R. (2024) Analysis of bacterial transcriptome and epitranscriptome using nanopore direct RNA sequencing.<strong><I> Nucleic Acids Res</strong></I>. 2024 DOI: 10.1093/nar/gkae601</span>		
            <br>
            <br>
            <span>89. He L, Xiong L, Wang X, Li Y, Li B, Shi Y, <strong>Deng X</strong>, Cui Z. (2024) Application of inhibitors targeting the type III secretion system in phytopathogenic bacteria.<strong><I> Chinese Chem Lett</strong></I>. 2024 DOI: 10.1016/j.cclet.2024.110044</span>		
            <br>
            <br>
            <span>88. Li P, Pan J, Dong Y, Sun Y, Wang Y, Liao K, Chen Y, <strong>Deng X</strong>, Yu S, Hu H. (2024) Microenvironment responsive charge-switchable nanoparticles act on biofilm eradication and virulence inhibition for chronic lung infection treatment.<strong><I> J Control Release</strong></I>. 2024 DOI: 10.1016/j.jconrel.2023.11.032</span>		
            <br>
            <br>
            <span>87. Huang J, Chen F, Lu B, Sun Y, Li Y, Hua C, <strong>Deng X<sup>*</sup></strong>. (2024) DNA Methylome Regulates Virulence and Metabolism in <I>Pseudomonas syringae</I>.<strong><I> eLife</strong></I>. 2024 DOI: 10.7554/eLife.96290.1</span>		
            <br>
            <br>
            <span>86. Sun Y, Li J, Huang J, Li S, Li Y, <strong>Deng X<sup>*</sup></strong>. (2024) Architecture of genome-wide transcriptional regulatory network reveals dynamic functions and evolutionary trajectories in <I>Pseudomonas syringae</I>.<strong><I> eLife</strong></I>. 2024 DOI: 10.7554/eLife.96172.1</span>		
            <br>
            <br>
            <span>85. Zhou J, Zhang H, Zhong K, Tao L, Lin Y, Xie G, Tan Y, Wu Y, Lu Y, Chen Z, Li J, <strong>Deng X</strong>, Peng Q, Li Z, Wang H. (2024) N6-methyladenosine facilitates mitochondrial fusion of colorectal cancer cells via induction of GSH synthesis and stabilization of OPA1 mRNA.<strong><I> Natl Sci Rev</strong></I>. 2024 DOI: 10.1093/nsr/nwae039</span>		
            <br>
            <br>
            <span>84. Li M, Guo H, Wang B, Han Z, Wu S, Liu J, Huang H, Zhu J, An F, Lin Z, Mo K, Tan J, Liu C, Wang L, <strong>Deng X</strong>, Li G, Ji J, Ouyang H. (2024) The single-cell transcriptomic atlas and RORA-mediated 3D epigenomic remodeling in driving corneal epithelial differentiation.<strong><I> Nat Commun</strong></I>. 2024 DOI: 10.1038/s41467-023-44471-w</span>		
            <br>
            <br>
            <h4 class="light-blue base-font marginTOP">20<span class="year-text">23</span></h4>
            <span>83. Shi Y, Xiong LT, Li H, Li WL, O'Neill Rothenberg D, Liao LS, <strong>Deng X</strong>, Song GP, Cui ZN. (2023) Derivative of cinnamic acid inhibits T3SS of <I>Xanthomonas oryzae</I> pv. oryzae through the HrpG-HrpX regulatory cascade.<strong><I> Bioorg Chem</strong></I>. 2023 DOI:10.1016/j.bioorg.2023.106871</span>		
            <br>
            <br>
            <span>82. Guo QQ, Li YZ, Shi HB, Yi AY, Xu XL, Wang HH, <strong>Deng X</strong>, Wu ZB, Cui ZN. (2023) Novel mandelic acid derivatives suppress virulence of  <I>Ralstonia solanacearum</I> via type III secretion system. <strong><I> Pest Manag Sci</strong></I>. 2023 DOI: 10.1002/ps.7664</span>		
            <br> 
            <br>
            <span>81. He LL, Wang X, O'Neill Rothenberg D, Xu X, Wang HH, <strong>Deng X</strong>, Cui ZN. (2023)A novel strategy to control  <I>Pseudomonas syringae</I> through inhibition of type III secretion system. <strong><I>Pestic Biochem Physiol</strong></I>. 2023 DOI:10.1016/j.pestbp.2023.105471 </span>
            <br>
            <br>
            <span>80. Wang T, Hua C,<strong>Deng X<sup>*</sup></strong>. (2023) c-di-GMP signaling in <I>Pseudomonas syringae</I>. <strong><I>Microbiol Res</strong></I>. 2023 DOI:10.1016/j.micres.2023.127445</span>			
            <br>
            <br>
            <span>79. Gao D, Li H, Shao J, He L, Fu C, Lai H, O'Neill Rothenberg D, Xu X, Song G, <strong>Deng X</strong>,Cui ZN.  (2023) Novel ethyl-3-aryl-2-nitroacrylate derivatives as potential T3SS inhibitors against <I>Xanthomonas oryzae</I> pv. oryzae on rice. <strong><I> J Agric Food Chem</strong></I>. 2023 DOI:10.1021/acs.jafc.3c00838</span>		
            <br>
            <br>
            <span>78. Ding Y, Li J, Li T, <strong>Deng X<sup>*</sup></strong>. (2023) Protocol for integrated analysis of bacterial RNA-seq and ChIP-seq data for establishing a co-expression network.  <strong><I>STAR Protoc</strong></I>. 2023 DOI:10.1016/j.xpro.2023.102322 </span>
            <br>
            <br>
            <span>77. Li JB, Xiong LT, Lu YR, Zhang YQ, Xu XL, Wang HH, <strong>Deng X</strong>,Hu XH, Cui ZN. (2023) Novel pyrimidin-4-one derivatives as potential T3SS inhibitors against <I>Xanthomonas campestris pv. campestris</I>.   <strong><I>Pest Manag Sci</strong></I>. 2023 DOI:10.1002/ps.7545 </span>			
            <br>
            <br>
            <span>76. Xu L, Xu X, Liang Y, Wen C, Ouyang K, Huang J, Xiao Y, <strong>Deng X</strong>,Xia J, Duan L. (2023) Osteoclast-targeted delivery of anti-miRNA oligonucleotides by red blood cell extracellular vesicles.   <strong><I>J Control Release</strong></I>. 2023 DOI:10.1016/j.jconrel.2023.04.043 </span>
            <br>
            <br>
            <span>75. Li K, Liao C, Wang T, Song L, <strong>Deng X</strong>,Yan W, Fang R, Jia Y, Tang X. (2023) Type III Secretion System Repressor RhpR Induces GrlP, a Glycine-Rich Outer Membrane Lipoprotein with Functions in Regulating the Periplasmic Space and Pleiotropic Responses.   <strong><I>Appl Environ Microbiol</strong></I>. 2023 DOI:10.1128/aem.01587-22 </span>
            <br>
            <br>
            <span>74. Fu XZ, Wu J, Li J, Ding J, Cui S, Wang XM, Wang YJ, Liu HQ, <strong>Deng X</strong>,Liu DF, Li WW. (2023) Heavy-metal resistant bio-hybrid with biogenic ferrous sulfide nanoparticles: pH-regulated self-assembly and wastewater treatment application.   <strong><I>J Hazard Mater</strong></I>. 2023 DOI:10.1016/j.jhazmat.2022.130667 </span>
            <h4 class="light-blue base-font marginTOP">20<span class="year-text">22</span></h4>
            <span>73. Huang J, Yao C, Sun Y, Ji Q, <strong>Deng X<sup>*</sup></strong>. (2022) Virulence-related regulatory network of <I>Pseudomonas syringae</I>.  <strong><I>Comput Struct Biotechnol J.</strong></I>. 2022 DOI:10.1016/j.csbj.2022.11.011. </span>
            <br>
            <br>
            <span>72. Shao X, Yao C, Ding Y, Qian G, He M, <strong>Deng X<sup>*</sup></strong>. (2022) The transcriptional regulators of virulence for <I>Pseudomonas aeruginosa</I>: Therapeutic opportunity and preventive potential of its clinical infections.  <strong><I>Genes & Diseases</strong></I>. 2022 DOI:10.1016/j.gendis.2022.09.009 </span>
            <br>
            <br>
            <span>71. Xie Y<sup>#</sup>, Li J<sup>#</sup>, Ding Y<sup>#</sup>, Shao X, Sun Y, Xie F, Liu S, Tang S, <strong>Deng X<sup>*</sup></strong>. (2022) An atlas of bacterial two-component systems reveals function and plasticity in signal transduction. <strong><I>Cell Reports</strong></I>. 2022 DOI: 10.1016/j.celrep.2022.111502 </span>
            <br>
            <br>
            <span>70. Hua C<sup>#</sup>, Huang J<sup>#</sup>, Wang T<sup>#</sup>, Sun Y, Liu J, Huang L, <strong>Deng X<sup>*</sup></strong>. (2022) Bacterial transcription factors bind to coding regions and regulate internal cryptic promoters. <strong><I>mBio</strong></I>. 2022 DOI: 10.1128/mbio.01643-22 </span>
            <br>
            <br>
            <span>69. Sun Y, Shao X, Zhang Y, Han L, Huang J, Liu J, <strong>Deng X<sup>*</sup></strong>. (2022) Maintenance of tRNA and Elongation Factors Supports T3SS Proteins Translational Elongations in Pathogenic Bacteria during Nutrient Starvation. <strong><I>Cell & Bioscience</strong></I>. 2022 DOI:10.1186/s13578-022-00884-6 </span>
            <br>
            <br>
            <span>68. Zou Y, Chen X, Sun Y, Li P, Xu M, Fang P, Zhang S, Yuan G, <strong>Deng X</strong>,Hu H. (2022) Antibiotics-free nanoparticles eradicate Helicobacter pylori biofilms and intracellular bacteria. <strong><I>Journal of Controlled Release</strong></I>. 2022 DOI:10.1016/j.jconrel.2022.05.044 </span>
            <br>
            <br>
            <span>67. Rao Y, Sun Y, Li P, Xu M, Chen X, Wang Y, Chen Y, <strong>Deng X</strong>,Yu S, Hu H. (2022) Hypoxia-sensitive adjuvant loaded liposomes enhance the antimicrobial activity of azithromycin via phospholipase-triggered releasing for <I>Pseudomonas aeruginosa</I> biofilms eradication. <strong><I>International Journal of Pharmaceutics</strong></I>. 2022 DOI:10.1016/j.ijpharm.2022.121910 </span>
            <br>
            <br>
            <span>66. Qin S, Xiao W, Zhou C, Pu Q, <strong>Deng X</strong>,Lan L, Liang H, Song X, Wu M. (2022) <I>Pseudomonas aeruginosa</I>: pathogenesis, virulence factors, antibiotic resistance, interaction with host, technology advances and emerging therapeutics. <strong><I>Signal Transduction and Targeted Therapy</strong></I>. 2022 DOI:10.1038/s41392-022-01056-1 </span>
            <br>
            <br>
            <span>65. Cai Z, Yang F, Shao X, Yue Z, Li Z, Song Y, Pan X, Jin Y, Cheng Z, Ha UH, Feng J, Yang L, <strong>Deng X</strong>,Wu W, Bai F. (2022) ECF sigma factor HxuI is critical for in vivo fitness of <I>Pseudomonas aeruginosa</I> during infection. <strong><I>Microbiology spectrum</strong></I>. 2022 DOI:10.1128/spectrum.01620-21 </span>
            <h4 class="light-blue base-font marginTOP">20<span class="year-text">21</span></h4>
            <span>64. Xie Y<sup>#</sup>, Ding Y<sup>#</sup>, Shao X, Yao C, Li J, Liu J, <strong>Deng X<sup>*</sup></strong>. (2021) <I>Pseudomonas syringae</I> senses polyphenols via phosphorelay crosstalk to inhibit virulence. <strong><I>EMBO Reports</strong></I>. 2021 DOI:10.15252/embr.202152805 </span>
            <br>
            <br>
            <span>63. Yao C<sup>#</sup>, Shao X<sup>#</sup>, Li J, <strong>Deng X<sup>*</sup></strong>. (2021) Optimized Protocols for ChIP-seq and deletion mutant construction in <I>Pseudomonas syringae</I> . <strong><I>STAR Protocols</strong></I>. 2021 DOI:10.1016/j.xpro.2021.100776
            <br>
            <br>
            <span>62. Po K, Chow H, Cheng Q, Chan B, <strong>Deng X</strong>,Wang S, Chan E, Kong H, Chan K, Li X, Chen S. (2021) Daptomycin exerts bactericidal effect through induction of excessive ROS production and blocking the function of stress response protein Usp2. <strong><I>Natural Sciences</strong></I>. 2021 DOI:10.1002/ntls.10023 </span>
            <br>
            <br>
            <span>61. Feng Z, Zhou F, Tan M, Wang T, Chen Y, Xu, W, Li B, Wang X<sup>*</sup>, <strong>Deng X<sup>*</sup></strong>,He M<sup>*</sup>. (2021) Targeting m6A modification inhibits herpes virus 1 infection. <strong><I>Genes & Diseases</strong></I>. 2021 DOI:10.1016/j.gendis.2021.02.004. </span>
            <br>
            <br>
            <span>60. Lu B, Yan Y, Dong L, Han L, Liu Y, Yu J, Chen J, Yi D, Zhang M,  <strong>Deng X</strong>,Wang C, Wang R, Wang D, Wei H, Liu D, Yi C. (2021) Integrated characterization of SARS-CoV-2 genome, microbiome, antibiotic resistance and host response from single throat swabs. <strong><I>Cell Discov</strong></I>. 2021 DOI:10.1038/s41421-021-00248-3. </span>
            <br>
            <br>
            <span>59. Shao X<sup>#</sup>, Tan M<sup>#</sup>, Xie Y, Yao C, Wang T, Huang H, Zhang Y, Ding Y, Liu J, Han L, Hua C, Wang X<sup>*</sup>, <strong>Deng X<sup>*</sup></strong>. (2021) Integrated regulatory network in <I>Pseudomonas syringae</I> reveals dynamics of virulence. <strong><I>Cell Reports</strong></I>. 2021 DOI:10(2). 34:108920. </span>
            <br>
            <br>
            <span>58. Wang T<sup>#</sup>, Sun W<sup>#</sup>, Fan L<sup>#</sup>, Hua C<sup>#</sup>, Wu N, Fan S, Zhang J, <strong>Deng X<sup>*</sup></strong>, Yan J<sup>*</sup>. (2021) An atlas of the binding specificities of transcription factors in <I>Pseudomonas aeruginosa</I> directs prediction of novel regulators in virulence. <strong><I>eLife</strong></I>. 2021 DOI:10:e61885. </span>
            <br>
            <h4 class="light-blue base-font marginTOP">20<span class="year-text">20</span></h4>
            <span>57. Fan L<sup>#</sup>, Wang T<sup>#</sup>, Hua C<sup>#</sup>, Sun W<sup>#</sup>, Li X, Grunwald L, Liu J, Wu N, Shao X, Yin Y, Yan J<sup>*</sup>, <strong>Deng X<sup>*</sup></strong>. (2020)  A Compendium of DNA-Binding Specificities of Transcription Factors in <I>Pseudomonas syringae</I>. <strong><I> Nature Commun</strong></I>. 2020 DOI:10.1038/s41467-020-18744-7. </span>
            <br>
            <br>
            <span>56. Chen G, Gan J, Yang C, Zuo Y, Peng J, Li M, Huo W, Xie Y, Zhang Y, Wang T, <strong>Deng X</strong>, Liang H. (2020)  The SiaA/B/C/D signaling network regulates biofilm formation in <I>Pseudomonas aeruginosa</I>. <strong><I>EMBO J</strong></I>. 2020 DOI:16;39(6):e103412.  </span>
            <br>
            <br>
            <span>55. Hou C, He W, Wang Z, Yi B, Hu Z, Wang W, <strong>Deng X</strong>, Yao X. (2020)  Particulate-Aggregated Adhesives with Exudate-Sensitive Properties and Sustained Bacteria Disinfection to Facilitate Wound Healing. <strong><I>ACS Appl Mater Interfaces</strong></I>. 2020 Jul 15. DOI: 12(28):31090-31098. </span>
            <br>
            <br>
            <span>54. Qin S, Li J, Zhou C, Privratsky B, Schettler J, <strong>Deng X</strong>, Xia Z, Zeng Y, Wu H, Wu M. (2020)  SHIP-1 Regulates Phagocytosis and M2 Polarization Through the PI3K/Akt-STAT5-Trib1 Circuit in <I>Pseudomonas aeruginosa</I> Infection. <strong><I> Front Immunol</strong></I>. 2020 Mar 18;11:307.  </span>
            <br>
            <br>
            <span>53. Xie Y, Liu W, Shao X, Zhang W, <strong>Deng X<sup>*</sup></strong>. (2020) Signal transduction schemes in <I>Pseudomonas syringae</I>. <strong><I>Comput Struct Biotechnol J</strong></I>. 2020. DOI: 10.1016/j.csbj.2020.10.039. </span>
            <br>
            <br>
            <span>52. Cao Q, Yang N, Wang Y, Xu C, Zhang X, Fan K, Chen F, Liang H, Zhang Y, <strong>Deng X</strong>, Feng Y, Yang C, Wu M, Bae T, Lan L. (2020)  Mutation-induced remodeling of the BfmRS two-component system in <I>Pseudomonas aeruginosa</I> clinical isolates. <strong><I>Sci Signal</strong></I>. 2020 Nov 3. DOI: 10.1126/scisignal.aaz1529. </span>
            <br>
            <br>
            <span>51. Shao X, Xie Y, Zhang Y, Liu J, Ding Y, Wu M, Wang X, <strong>Deng X<sup>*</sup></strong>. (2020)  Novel therapeutic strategies for treating <I>Pseudomonas aeruginosa</I> infection. <strong><I>Expert Opinion on Drug Discovery</strong></I>. 2020. DOI: 10.1080/17460441.2020.1803274. </span>
            <br>
            <br>
            <span>50. Hua C, Wang T, Shao X, Xie Y, Huang H, Liu J, Zhang W, Zhang Y, Ding Y, Jiang L, Wang X, <strong>Deng X<sup>*</sup></strong>. (2020) <I>Pseudomonas syringae</I> dual-function protein Lon switches between virulence and metabolism by acting as both DNA-binding transcriptional regulator and protease in different environments. <strong><I>Environ Microbiol</strong></I>. 2020. doi.org/10.1111/1462-2920.15067. </span>
            <br>
            <br>
            <span>49. Shao X<sup>#</sup>, Zhang W<sup>#</sup>, Umar M<sup>#</sup>, Wong H, Seng Z, Xie Y, Zhang Y, Yang L, Kwok C<sup>*</sup>, <strong>Deng X<sup>*</sup></strong>. (2020) RNA G-quadruplexes structures mediate gene regulation in bacteria. <strong><I>mBio</strong></I>. 2020 Jan 21; 10(2). 11(1): e02926-19. </span>
            <br>
            <h4 class="light-blue base-font marginTOP">201<span class="year-text">9</span></h4>
            <span>48. Yang Z, Yang Y, Wang M, Wang T, Fok HK, Jiang B, Xiao W, Kou S, Guo Y, Yan Y, <strong>Deng X</strong>, Zhang WB, Sun F. Dynamically Tunable, Macroscopic Molecular Networks Enabled by Cellular Synthesis of 4-Arm Star-like Proteins. <strong><I>Matter</strong></I>. 2019 Oct 9.</span>
            <br>
            <br>
            <span>47. Xie Y, Shao X, <strong>Deng X<sup>*</sup></strong>. (2019) Regulation of Type III Secretion System in <I>Pseudomonas syringae</I>. <strong><I>Environ Microbiol</strong></I>. 2019 Aug 13. doi: 10.1111/1462-2920.14779.</span>
            <br>
            <br>
            <span>46. Zhang Y <sup>#</sup>, Zhou C <sup>#</sup>, Pu Q, Wu Q, Tan S, Shao X, Zhang W, Xie Y, Li R, Yu X, Wang R, Zhang L, Wu M<sup>*</sup>, <strong>Deng X<sup>*</sup></strong>. (2019) <I>Pseudomonas aeruginosa</I> regulatory protein AnvM controls pathogenicity in anaerobic environments and impacts host defense. <strong><I>mBio</strong></I>. 2019 Jul 23; 10(4). pii: e01362-19. </span>
            <br>
            <br>
            <span>45. Huang H<sup>#</sup>, Shao X<sup>#</sup>, Xie Y<sup>#</sup>, Wang T, Zhang Y, Wang X<sup>*</sup>, <strong>Deng X<sup>*</sup></strong>. (2019) An integrated genomic regulatory network of virulence-related transcriptional factors in <I>Pseudomonas aeruginosa</I>. <strong><I>Nat Commun</strong></I>. 10: 2931. 
            <br>
            <a class="text-underline" href="https://www.sciencedaily.com/releases/2019/07/190703121423.htm">https://www.sciencedaily.com/releases/2019/07/190703121423.htm</a>
            <br>
            <a class="text-underline" href="https://www.news-medical.net/news/20190704/Researchers-reveal-virulence-regulatory-mechanism-in-a-superbug.aspx">https://www.news-medical.net/news/20190704/Researchers-reveal-virulence-regulatory-mechanism-in-a-superbug.aspx</a>
            </span>
            <br>
            <br>
            <span>44. Shao X, Xie Y, Zhang Y, <strong>Deng X<sup>*</sup></strong>. (2019) Biofilm Formation Assay in <I>Pseudomonas syringae</I>. <strong><I>Bio-protocol</strong></I>. 9(10): e3237. </span>
            <br>
            <br>
            <span>43. Kandimalla R, Gao F, Li Y, Huang H, Ke J, <strong>Deng X</strong>, Zhao L, Zhou S, Goel A, Wang X. (2019) RNAMethyPro: a biologically conserved signature of N6-methyladenosine regulators for predicting survival at pan-cancer level. <strong><I>NPJ Precis Oncol</strong></I>. 3: 13. </span>
            <br>
            <br>
            <span>42. Peng C, Huang D, Shi Y, Zhang B, Sun L, Li M, <strong>Deng X</strong>, Wang W. (2019) Comparative transcriptomic analysis revealed the key pathways responsible for organic sulfur removal by thermophilic bacterium <I>Geobacillus thermoglucosidasius</I> W-2. <strong><I>Sci Total Environ</strong></I>. 676: 639-650. </span>
            <br>
            <br>
            <span>41. Li K, Zhu Y, Yan W, <strong>Deng X</strong>, Xiao Y, Song L, Fang R, Jia Y, Tang X. (2019) Two components of the <I>rhpPC</I> operon coordinately regulate the type III secretion system and bacterial fitness in <I>Pseudomonas savastanoi</I> pv. <I>phaseolicola</I>. <strong><I>PLoS Pathog</strong></I>. 15(4): e1007673. </span>
            <br>
            <br>
            <span>40. Xie Y, Shao X, Zhang Y, Liu J, Wang T, Zhang W, Hua C, <strong>Deng X<sup>*</sup></strong>. (2019) <I>Pseudomonas savastanoi</I> two-component system RhpRS switches between virulence and metabolism by tuning phosphorylation state and sensing nutritional conditions. <strong><I>mBio</strong></I>. 2019 Mar 19; 10(2). pii: e02838-18. </span>
            <br>
            <br>
            <span>39. Wang T<sup>#</sup>, Cai Z<sup>#</sup>, Shao X, Zhang W, Xie Y, Zhang Y, Hua C, Schuster S C, Yang L, <strong>Deng X<sup>*</sup></strong>. (2019) The Pleiotropic Effects of c-di-GMP Content in <I>Pseudomonas syringae</I>. <strong><I>Appl Environ Microbiol</strong></I>. 2019 Mar 8. pii: AEM.00152-19. </span>
            <br>
            <br>
            <span>38. Sun L, Huang D, Zhu L, Zhang B, Peng C, Ma T, <strong>Deng X</strong>, Wu J, Wang W. (2019) Novel thermostable enzymes from <I>Geobacillus thermoglucosidasius</I> W-2 for high-efficient nitroalkane removal under aerobic and anaerobic conditions. <strong><I>Bioresour Technol</strong></I>. 2019. 278: 73-81. </span>
            <br>
            <h4 class="light-blue base-font marginTOP">201<span class="year-text">8</span></h4>
            <span>37. Shao X<sup>#</sup>, Zhang X<sup>#</sup>, Zhang Y, Zhu M, Yang P, Yuan J, Xie Y, Zhou T, Wang W, Chen S, Liang H<sup>*</sup>, <strong>Deng X<sup>*</sup></strong>. (2018) RpoN-dependent direct regulation of quorum sensing and the type VI secretion system in <I>Pseudomonas aeruginosa</I> PAO1. <I><strong>J Bacteriol</strong></I>. 2018. 200(16). pii: e00205-18. </span>
            <br>
            <br>
            <span>36. Wang J<sup>#</sup>, Shao X<sup>#</sup>, Zhang Y, Zhu Y, Yang P, Yuan J, Wang T, Yin C, Wang W, Chen S, Liang H<sup>*</sup>, <strong>Deng X<sup>*</sup></strong>. (2018) HrpS is a global regulator on T3SS and non-T3SS genes in <I>Pseudomonas savastanoi</I> pv. <I>phaseolicola</I>. <I><strong>Mol Plant Microbe Interact</strong></I>. 2018. 31(12): 1232-1243. </span>
            <br>
            <h4 class="light-blue base-font marginTOP2">201<span class="year-text">7</span></h4>
            <span>35. Ye Y, Lin P, Zhang W, Tan S, Zhou X, Li R, Pu Q, Koff JL, Dhasarathy A, Ma F, <strong>Deng X<sup>*</sup></strong>, Jiang J<sup>*</sup>, Wu M<sup>*</sup>. (2017) DNA repair interacts with autophagy to regulate inflammatory responses to pulmonary hyperoxia. <I><strong>J Immunol</strong></I>.  198 (7) 2844-2853.</span>
            <br>
            <br>
            <span>34. Pu Q, Gan C, Li R, Yi L, Tan S, Li X, Wei Y, Lan L, <strong>Deng X</strong>, Liang H, Ma F, Wu M. (2017) Atg7 deficiency intensifies inflammasome activation and pyroptosis in <I>Pseudomonas</I> sepsis. <I><strong>J Immunol</strong></I>. 198(8), 3205.</span>
            <br>
            <h4 class="light-blue base-font marginTOP2">201<span class="year-text">6</span></h4>
            <span>33. Zhou T, Yin C, Zhang Y, Shi H, Wang J, Sun L, Shao X, Gao R, Wang W<sup>*</sup>, <strong>Deng X<sup>*</sup></strong>. (2016) Lon protease is involved in RhpRS-mediated regulation of type III secretion in <I>Pseudomonas syringae</I>. <I><strong>Mol Plant Microbe Interact</strong></I>. 29(10): 807-814.</span>
            <br>
            <br>
            <span>32. Zhao J, Yu X, Zhu M, Kang H, Ma J, Wu M, Gan J<sup>*</sup>, <strong>Deng X<sup>*</sup></strong> and Liang H<sup>*</sup>. (2016) Structural and molecular mechanism of CdpR involved in quorum-sensing and bacterial virulence in <I>Pseudomonas aeruginosa</I>. <I><strong>PLOS Biol</strong></I>. 14(4): e1002449.</span>
            <br>
            <br>
            <span>31. Yu P, Yuan J, Zhang H, <strong>Deng X</strong>, Ma M, Zhang H. (2016) Engineering metal-binding sites of bacterial CusF to enhance Zn/Cd accumulation and resistance by subcellular targeting. <I><strong>J Hazard Mater</strong></I>. 302: 275-85.</span>
            <br>
            <br>
            <span>30. Yu P, Yuan J, <strong>Deng X</strong>, Ma M, Zhang H. (2016) Subcellular targeting of bacterial CusF enhances Cu accumulation and alters root to shoot Cu translocation in <I>Arabidopsis</I>. <I><strong>Plant Cell Physiol</strong></I>. 55(9): 1568-81.</span>
            <br>
            <br>
            <span>29. Luo GZ, Wang F, Weng X, Chen K, Hao Z, Yu M, <strong>Deng X</strong>, Liu J, He C. (2016) Characterization of eukaryotic DNA N<sup>6</sup>-methyladenine by a highly sensitive restriction enzyme-assisted sequencing. <I><strong>Nat Commun</strong></I>. 7: 11301.</span>
            <br>
            <h4 class="light-blue base-font marginTOP2">201<span class="year-text">5</span></h4>
            <span>28. Kong W, Zhao J, Zhu M, Kang H, Zhou T, <strong>Deng X<sup>*</sup></strong>, Liang H<sup>*</sup>. (2015) ChIP-seq reveals molecular mechanisms of global regulator AlgR mediating cyclic di-GMP synthesis and biofilm formation in <I>Pseudomonas aeruginosa</I>. <I><strong>Nucleic Acids Res</strong></I>. 43(17): 8268-82.</span>
            <br>
            <br>
            <span>27. <strong>Deng X<sup>#*</sup></strong>, Chen K<sup>#</sup>, Luo GZ, Weng X, Ji Q, Zhou T, He C<sup>*</sup>. (2015) Widespread occurrence of N<sup>6</sup>-methyladenosine in bacterial mRNA. <I><strong>Nucleic Acids Res</strong></I>. 43(13): 6557-67.</span>
            <br>
            <br>
            <span>26. Zhou T, Chen K, Zhang HX, <strong>Deng X<sup>*</sup></strong>. (2015) Genome-wide DNA binding pattern of two-component system response regulator RhpR in <I>Pseudomonas syringae</I>. <I><strong>Genomics Data</strong></I>. (4): 146-7.</span>
            <br>
            <br>
            <span>25. Fu Y, Luo GZ, Chen K, <strong>Deng X</strong>, Yu M, Han D, Hao Z, Liu J, Lu X, Dore LC, Weng X, Ji Q, Mets L, He C. (2015) N<sup>6</sup>-methyldeoxyadenosine marks active transcription start sites in <I>Chlamydomonas</I>. <I><strong>Cell</strong></I>. 161(4): 879-92.</span>
            <br>
            <br>
            <span>24. Yang N, Ding S, Chen F, Zhang X, Xia Y, Di H, Cao Q, <strong>Deng X</strong>, Wu M, Wong CC, Tian XX, Yang CG, Zhao J, Lan L. (2015) The Crc protein participates in down-regulation of the Lon gene to promote rhamnolipid production and <I>rhl</I> quorum sensing in <I>Pseudomonas aeruginosa</I>. <I><strong>Mol Microbiol</strong></I>. 96(3): 526-47.</span>
            <br>
            <h4 class="light-blue base-font marginTOP2">201<span class="year-text">4</span></h4>
            <span>23. <strong>Deng X#</strong>, Liang H#, Chen, K, He C, Lan L, Tang X. (2014) Molecular mechanisms of two-component system RhpRS regulating type III secretion system in <I>Pseudomonas syringae</I>. <I><strong>Nucleic Acids Res</strong></I>. 42(18): 11472-86.</span>
            <br>
            <br>
            <span>22. Liang H<sup>#</sup>, <strong>Deng X<sup>#</sup></strong>, Li X, Ye Y, Wu M. (2014) Molecular mechanisms of master regulator VqsM mediating quorum-sensing and antibiotic resistance in <I>Pseudomonas aeruginosa</I>. <I><strong>Nucleic Acids Res</strong></I>. 42 (16): 10307-10320.</span>
            <br>
            <br>
            <span>21. <strong>Deng X</strong>, Liang H, Ulanovskaya O, Ji Q, Zhou T, Sun F, Lu Z, Hutchison A, Lan L, Wu M, Cravatt B, He C. (2014) Steady-state hydrogen peroxide induces glycolysis in <I>Staphylococcus aureus</I> and <I>Pseudomonas aeruginosa</I>. <I><strong>J Bacteriol</strong></I>. 196(14): 2499-2513.</span>
            <br>
            <br>
            <span>20. Ding Y, Liu X, Chen F, Di H, Xu B, Zhou L, <strong>Deng X</strong>, Wu M, Yang CG, Lan L. (2014) Metabolic sensor governing bacterial virulence in <I>Staphylococcus aureus</I>. <I><strong>Proc Natl Acad Sci U S A</strong></I>.111(46): E4981-90.</span>
            <br>
            <br>
            <span>19. Cao Q, Wang Y, Chen F, Xia Y, Lou J, Zhang X, Yang N, Sun X, Zhang Q, Zhuo C, Huang X, <strong>Deng X</strong>, Yang CG, Ye Y, Zhao J, Wu M, Lan L. (2014) A novel signal transduction pathway that modulates <I>rhl</I> quorum sensing and bacterial virulence in <I>Pseudomonas aeruginosa</I>. <I><strong>PLOS Pathog</strong></I>. 10(8): e1004340.</span>
            <br>
            <br>
            <span>18. Yu P, Yuan J, <strong>Deng X</strong>, Ma M, Zhang H. (2014) Subcellular targeting of bacterial CusF enhances Cu accumulation and alters root to shoot Cu translocation in <I>Arabidopsis</I>. <I><strong>Plant Cell Physiol</strong></I>. 55 (9): 1568-1581.</span>
            <br>
            <br>
            <span>17. Liu J, Yue Y, Han D, Wang X, Fu Y, Zhang L, Jia G, Yu M, Lu Z, <strong>Deng X</strong>, Dai Q, Chen W, He C. (2014) A METTL3-METTL14 complex mediates mammalian nuclear RNA N<sup>6</sup>-adenosine methylation. <I><strong>Nat Chem Biol</strong></I>. 10: 93-95.</span>
            <br>
            <h4 class="light-blue base-font marginTOP2">201<span class="year-text">3</span></h4>
            <span>16. <strong>Deng X</strong>, Weerapana E, Ulanovskaya O, Sun F, Liang H, Ji Q, Ye Y, Fu Y, Zhou L, Li J, Zhang H, Wang C, Alvarez S, Hicks ML, Lan L, Wu M, Cravatt B, He C. (2013) Proteome-wide quantification and characterization of oxidation-sensitive cysteines in pathogenic bacteria. <I><strong>Cell Host Microbe</strong></I>. 13 (3): 358-70.</span>
            <br>
            <br>
            <span>15. Ji Q, Zhang L, Jones MB, Sun F, <strong>Deng X</strong>, Liang H, Cho H, Brugarolas P, Gao NY, Peterson NS, Lan L, Bae T, He C. (2013) Molecular mechanism of quinone signaling mediated through S-quinonization of a YodB family repressor QsrR. <I><strong>Proc Natl Acad Sci U S A</strong></I>. 110(13): 5010-5.</span>
            <br>
            <br>
            <span>14. Liang H, <strong>Deng X</strong>, Bosscher M, Ji Q, Jensen MP, He C. (2013) Engineering bacterial two-component system PmrA/PmrB to sense lanthanide ions. <I><strong>J Am Chem Soc</strong></I>. 135(6):2037-9.</span>
            <br>
            <h4 class="light-blue base-font marginTOP2">201<span class="year-text">2</span></h4>
            <span>13. Liang H<sup>#</sup>, <strong>Deng X<sup>#</sup></strong>, Ji Q, Sun F, Shen T, He C. (2012) The <I>Pseudomonas aeruginosa</I> global regulator VqsR directly inhibits QscR to control quorum-sensing and virulence gene expression. <I><strong>J Bacteriol</strong></I>. 194(12): 3098-108.</span>
            <br>
            <br>
            <span>12. <strong>Deng X</strong>, Sun F, Ji Q, Liang H, Missiakas D, Lan L, He C. (2012) Expression of multidrug resistance efflux pump gene <I>norA</I> is iron-responsive in <I>Staphylococcus aureus</I>. <I><strong>J Bacteriol</strong></I>. 194(7): 1753-62.</span>
            <br>
            <br>
            <span>11. Sun F, Ding Y, Ji Q, Liang Z, <strong>Deng X</strong>, Wong CC, Yi C, Zhang L, Xie S, Alvarez S, Hicks LM, Luo C, Jiang H, Lan L, He C. (2012) Protein cysteine phosphorylation of SarA/MgrA family transcriptional regulators mediates bacterial virulence and antibiotic resistance. <I><strong>Proc Natl Acad Sci U S A</strong></I>. 109(38): 15461-6.</span>
            <br>
            <br>
            <span>10. Sun F, Liang H, Kong X, Xie S, Cho H, <strong>Deng X</strong>, Ji Q, Zhang H, Alvarez S, Hicks LM, Bae T, Luo C, Jiang H, He C. (2012) Quorum-sensing agr mediates bacterial oxidation response via an intramolecular disulfide redox switch in the response regulator AgrA. <I><strong>Proc Natl Acad Sci U S A</strong></I>. 109(23): 9095-100.</span>
            <br>
            <br>
            <span>9. Ji Q, Zhang L, Sun F, <strong>Deng X</strong>, Liang H, Bae T, He C. (2012) <I>Staphylococcus aureus</I> CymR is a new thiol-based oxidation-sensing regulator of stress resistance and oxidative response. <I><strong>J Biol Chem</strong></I>. 287(25): 21102-9.</span>
            <br>
            <br>
            <span>8. Qian Y, Zhang L, Ding S, <strong>Deng X</strong>, He C, Zheng EX, Zhu HL, Zhao J. A fluorescent probe for rapid detection of hydrogen sulfide in blood plasma and brain tissues in mice. (2012) <I><strong>Chem. Sci</strong></I>. 3: 2920-2923.</span>
            <br>
            <br>
            <span>7. Sun F, Ji Q, Jones MB, <strong>Deng X</strong>, Liang H, Frank B, Telser J, Peterson SN, Bae T, He C. (2012) AirSR, a [2Fe-2S] cluster-containing two-component system, mediates global oxygen sensing and redox signaling in <I>Staphylococcus aureus</I>. <I><strong>J Am Chem Soc</strong></I>. 134(1): 305-14.</span>
            <br>
            <h4 class="light-blue base-font marginTOP2">Before 201<span class="year-text">1</span></h4>
            <span>6. Sun F, Zhou L, Zhao BC, <strong>Deng X</strong>, Cho H, Yi C, Jian X, Song CX, Luan CH, Bae T, Li Z, He C. (2011) Targeting MgrA-mediated virulence regulation in <I>Staphylococcus aureus</I>. <I><strong>Chem Biol</strong></I>. 18(8): 1032-41.</span>
            <br>
            <br>
            <span>5. <strong>Deng X</strong>, Lan L, Kennelly M, Zhou JM, Tang X. (2010) <I>Pseudomonas syringae</I> two-component response regulator RhpR regulates genes under an inverted repeat element promoter. <I><strong>Mol Plant Microbe Interact</strong></I>. 23(7): 927-39.</span>
            <br>
            <br>
            <span>4. <strong>Deng X</strong>, Lan L, Xiao Y, Zhou JM, Tang X. (2009) <I>Pseudomonas syringae</I> pv. <I>phaseolicola</I> mutants compromising type III secretion system. <I><strong>Mol Plant Microbe Interact</strong></I>. 22(8): 964-76.</span>
            <br>
            <br>
            <span>3. Lan L, <strong>Deng X</strong>, Xiao Y, Zhou JM, Tang X. (2007) Mutation of Lon protease differentially affects the expression of <I>Pseudomonas syringae</I> type III secretion system genes in rich and minimal media and reduces pathogenicity. <I><strong>Mol Plant Microbe Interact</strong></I>. 20(6): 682-96.</span>
            <br>
            <br>
            <span>2. Xiao Y, Lan L, Yin C, <strong>Deng X</strong>, Baker D, Zhou JM, Tang X. (2007) Two-component sensor RhpS promotes induction of <I>Pseudomonas syringae</I> type III secretion system by repressing negative regulator RhpR. <I><strong>Mol Plant Microbe Interact</strong></I>. 20(3): 223-34.</span>
            <br>
            <br>
            <span>1. Lan L, <strong>Deng X</strong>, Zhou JM, Tang X. (2006) Genome-wide gene expression analysis of <I>Pseudomonas syringae</I> pv. <I>tomato</I> DC3000 reveals overlapping and distinct pathways regulated by <I>hrpL</I> and <I>hrpRS</I>. <I><strong>Mol Plant Microbe Interact</strong></I>. 19(9): 976-8.</span>
            <br>
            <br>
            <br>
        </div>
    </div>
</div>
<!-- FOOTER AREA -->
<div class="full-width footer-area">
    <div class="row">
        <div class="large-12 columns foot-posi">
            &copy; 2018 DENG LAB. All rights reserved.
        </div>
    </div>
</div>
</body>
</html>
